.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @knshnb @c-bata @contramundum53 @himkt @HideakiImamura @g-votte @hvy @iwiwi @not522 @sile @smly @toshihikoyanase @ytsmiling
+* @HideakiImamura @c-bata @contramundum53 @g-votte @himkt @hvy @iwiwi @knshnb @not522 @sile @smly @toshihikoyanase @ytsmiling
```

Comment:
I don't know why this CODEOWNERS file contains errors, but it is consistent with [recipe/meta.yml.](https://github.com/conda-forge/optuna-feedstock/blob/main/recipe/meta.yaml#L52)